bitrise.yml

```diff
@@ -373,6 +373,56 @@ workflows:
               exit 1
             fi
 
+  stack-versions:
```

Comment:
please add a prefix `test-`, thisworkflow name is not describing what will happen